pkg/mobile/metrics/fh-sync-server.go

```diff
@@ -0,0 +1,347 @@
+package metrics
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"time"
+
+	"bytes"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/feedhenry/mcp-standalone/pkg/mobile"
+	"github.com/pkg/errors"
+)
+
+// TODO: Add multi-worker support?
+type statsResponse struct {
+	Metrics struct {
+		CPUUsage struct {
```

Comment:
As these usage structs are the same, they can be abstracted

```
type Usage struct{
 Current string `json:"current"`
}

type CPUUsage struct{
    Master *Usage `json:"master"`
}

```

etc. Can discuss further in person

Reply:
Agreed :+1:  It can be simplified a lot into just 4 structs that are re-used